### PR TITLE
doc(README): add link to how to wiki instructions

### PR DESCRIPTION
**Pull request by corinnekrych**
I've added wiki instructions + link on readme for instruction on how to work with a local WIT and a local fabric8-ui

### CODE REVIEW

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2212-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2212-fabric8-ui.badger.fabric8.io

File: README.md [hunk 1]
```diff
@@ -21,18 +22,20 @@ There are others too. For example if you want to try [run fabric8 locally on min
 source environments/local-cluster.sh
 ```
 
+> NOTE: If you want to target a local WIT backend, check your wiki [How To pages](https://github.com/fabric8-ui/fabric8-ui/wiki/How-to...#how-to-run-local-fabric8-ui-with-local-wit).
```
Comment (joshuawilson) [inline README.md]: `check your wiki` should be `check the wiki` or `check our wiki`

**Comment by corinnekrych** [comment]
@joshuawilson thank for the review, I've juts updated it.

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2212-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2212-fabric8-ui.badger.fabric8.io

**Pull request merged**